### PR TITLE
Fixed the broken CI tests and the missing dependency issue

### PR DESCRIPTION
**Pull request by barhun**

### CODE REVIEW

File: .ci/run-app.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ APP_IMG_TAG=$(tr '[:upper:]' '[:lower:]' <<< $TEST_APP)_${TEST_PORT}
 .ci/build-app-image.sh $BASE_IMAGE $TEST_APP $APP_IMG_TAG $VERSION
 
 # Start app
-# .ci/start-container.sh 5004 $TEST_PORT $APP_IMG_TAG $APP_IMG_TAG
+.ci/start-container.sh 5004 $TEST_PORT $APP_IMG_TAG $APP_IMG_TAG
 
 echo "[CI] Verifying connectivity..."
-# .ci/test-connection.sh http://localhost:$TEST_PORT
+.ci/test-connection.sh http://localhost:$TEST_PORT
```
Comment (ahmetalpbalkan) [inline .ci/run-app.sh]: Wow can't believe I checked these in commented out. 😬

Comment (barhun) [inline .ci/run-app.sh]: Sorun degil, hocu. (:

**Comment by ahmetalpbalkan** [comment]
CI is failing  building the beta5 image with error

    Architecture must be x86 or x64.

Does anyone have any ideas how beta5 installation scripts have broken in the meanwhile. I thought we are using git version tags everywhere.

**Comment by barhun** [comment]
It must be all fixed now. Pending build results, I need to sleep then. (:

**Comment by ahmetalpbalkan** [comment]
LGTM.

**Pull request merged**